<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostAd</class>
 <widget class="QDialog" name="PostAd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1">
    <widget class="QLineEdit" name="editTags"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="editPrice"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Images</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Ad Title</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Contacts</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="editTitle"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelErrors">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red</string>
     </property>
     <property name="text">
      <string>Errors here</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Post an ad to the CraigsCoin blockchain</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QPlainTextEdit" name="editContacts">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText" stdset="0">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="cbxCountry">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxState"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbxTown"/>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="editAddress">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Zip Code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editZipCode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPlainTextEdit" name="editMessage">
     <property name="placeholderText" stdset="0">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="cbxCategory"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbxSubCategory"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="editName"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Youtube</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelBodyLength">
         <property name="text">
          <string>20/1000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPostFee">
         <property name="text">
          <string>Fee: 10</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QLabel" name="labelWarn">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Make sure to double check your ad:&lt;br/&gt;Nobody will be able to edit or delete your ad once you click &amp;quot;OK&amp;quot;, even you.&lt;br/&gt;By clicking &amp;quot;OK&amp;quot; you confirm that your ad does not violate your country laws.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QListWidget" name="listVideos">
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
       <item>
        <widget class="QToolButton" name="btnVideoAdd">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/add</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnVideoRemove">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/remove</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="11" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QListWidget" name="listImages">
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
       <item>
        <widget class="QToolButton" name="btnImageAdd">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/add</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnImageRemove">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/remove</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editContacts</tabstop>
  <tabstop>cbxCountry</tabstop>
  <tabstop>cbxState</tabstop>
  <tabstop>cbxTown</tabstop>
  <tabstop>editAddress</tabstop>
  <tabstop>editZipCode</tabstop>
  <tabstop>cbxCategory</tabstop>
  <tabstop>cbxSubCategory</tabstop>
  <tabstop>editTitle</tabstop>
  <tabstop>editMessage</tabstop>
  <tabstop>editTags</tabstop>
  <tabstop>editPrice</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PostAd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PostAd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editContacts</sender>
   <signal>textChanged()</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxTown</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editZipCode</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxSubCategory</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMessage</sender>
   <signal>textChanged()</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editTags</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editPrice</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxCountry</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxState</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxCategory</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>636</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listImages</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listVideos</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>PostAd</receiver>
   <slot>updateTextLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateTextLength()</slot>
 </slots>
</ui>
